### PR TITLE
Avoid using find which is not supported in IE11

### PR DESCRIPTION
**Pull request by johnBartos**
### Why is this Pull Request needed?
To prevent exceptions from being thrown in IE

### Are there any points in the code the reviewer needs to double check?
No

### Are there any Pull Requests open in other repos which need to be merged with this?
No

#### Addresses Issue(s):

JW8-612

### CODE REVIEW

File: dist/hls.js [hunk 1]
```diff
@@ -6267,6 +6262,22 @@ var playlist_loader_PlaylistLoader = function (_EventHandler) {
   return PlaylistLoader;
 }(event_handler);
 
+function findGroup() {
+  var groups = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : [];
+  var mediaGroupId = arguments[1];
```
Comment (robwalch) [inline dist/hls.js]: Why not define these parameters and default `groups = []`?

Comment (johnBartos) [reply dist/hls.js]: you're looking at the dist

Comment (johnBartos) [reply dist/hls.js]: I always do the real work in a separate commit: https://github.com/jwplayer/hls.js/pull/114/commits/b4aecd409b860fbf00598070e85d91a53d204e4c

Comment (robwalch) [reply dist/hls.js]: derp!

**Pull request merged**